blockchain/tx_pool_test.go

```diff
@@ -708,7 +708,7 @@ func TestTransactionDropping(t *testing.T) {
 	pool, key := setupTxPool()
 	defer pool.Stop()
 
-	account, _ := deriveSender(transaction(0, 0, key))
+	account := crypto.PubkeyToAddress(key.PublicKey)
```

Comment:
Does it support klaytn key type?

Reply:
The `key` type is ecdsa.privateKey, so it is irrelevant with the klaytn key(account) type. That is, only ecdsa.privateKey type is used in test.

Reply:
Make sense. Thanks for hard working~!